storage/src/mmr/proof.rs

```diff
@@ -317,15 +315,14 @@ impl<D: Digest> Proof<D> {
         &self,
         hasher: &mut H,
         elements: &[E],
-        start_element_pos: u64,
+        start_loc: u64,
```

Comment:
I think we're still passing in a position rather than location to this function in `adb::current::verify_range_proof` and `adb::current::verify_key_value_proof`.

Reply:
yeah the stuff outside the mmr module is currently WIP